Missing Converter:: in definition of kEtsiTypesParamSupportedOptions

Hi,
Thanks a lot for your great work on this repo! While going through the code, I noticed a small issue in one of the constant definitions that caused a compilation error on my side. In the following line:
```const std::vector<std::string> kEtsiTypesParamSupportedOptions{"cam", "cam_ts", "cpm_ts", "denm", "denm_ts", "mapem_ts", "spatem_ts", "vam_ts"};```
The line is missing the Converter:: scope. To avoid compilation errors, it should be:
```const std::vector<std::string> Converter::kEtsiTypesParamSupportedOptions{...};```
Just wanted to let you know in case you'd like to fix it 🙂

Best regards,  
Xiaoyue

Thanks for your quick response. After looking into it more carefully, I realized that this isn't actually a compilation error, but a runtime error.
When running:
```ros2 launch etsi_its_conversion converter.launch.py```
I ran into the following error:
```
[etsi_its_conversion_node-1]   what():  Could not load library dlopen error: /home/.../etsi_its_conversion/lib/libetsi_its_conversion.so: undefined symbol: _ZN19etsi_its_conversion9Converter31kEtsiTypesParamSupportedOptionsB5cxx11E, at ./src/shared_library.c:99 
[ERROR] [etsi_its_conversion_node-1]: process has died [pid 87110, exit code -6]
```
Hope this helps, please let me know if I can provide any more details.

--- a/etsi_its_conversion/etsi_its_conversion/src/Converter.cpp
+++ b/etsi_its_conversion/etsi_its_conversion/src/Converter.cpp
@@ -102,8 +102,8 @@ const std::string Converter::kEtsiMessagePayloadOffsetParam{"etsi_message_payloa
 const int Converter::kEtsiMessagePayloadOffsetParamDefault{78};
 const std::string Converter::kRos2UdpEtsiTypesParam{"ros2udp_etsi_types"};
 const std::string Converter::kUdp2RosEtsiTypesParam{"udp2ros_etsi_types"};
-const std::vector<std::string> kEtsiTypesParamSupportedOptions{"cam", "cam_ts", "cpm_ts", "denm", "denm_ts", "mapem_ts", "spatem_ts", "vam_ts"};
-const std::vector<std::string> Converter::kRos2UdpEtsiTypesParamDefault = kEtsiTypesParamSupportedOptions;
+const std::vector<std::string> Converter::kEtsiTypesParamSupportedOptions{"cam", "cam_ts", "cpm_ts", "denm", "denm_ts", "mapem_ts", "spatem_ts", "vam_ts"};
+const std::vector<std::string> Converter::kRos2UdpEtsiTypesParamDefault = Converter::kEtsiTypesParamSupportedOptions;
 const std::vector<std::string> Converter::kUdp2RosEtsiTypesParamDefault{"cam", "cpm_ts", "denm", "mapem_ts", "spatem_ts", "vam_ts"};
 const std::string Converter::kSubscriberQueueSizeParam{"subscriber_queue_size"};
 const int Converter::kSubscriberQueueSizeParamDefault{10};

